cypress/integration/sGMR/user-register.spec.js

```diff
@@ -107,4 +107,20 @@ describe('User Registration', () => {
       cy.url().should('include', '/sign-in?source=registration');
     });
   });
+
+  it('Should not register If Password does not meet validation rules', () => {
```

Comment:
`If Password` doesn't have to be uppercased.
```suggestion
  it('Should not register if password does not meet validation rules', () => {
```